Workshops/250410_CLEBIC_HyperSpy-LumiSpy.ipynb

```diff
@@ -0,0 +1,766 @@
+   "execution_count": null,
+   "id": "76b9df93-3636-4ad7-85c0-4e25a5f31e21",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "integrated_sliced_signal1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "677eccd1-367e-4740-8c7b-e1a744235e5e",
+   "metadata": {},
+   "source": [
+    "## Streak image fitting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2cc8bab-9401-45b2-8eb7-1eec46d04347",
+   "metadata": {},
+   "source": [
+    "*(In the following, we will use the dataset `streak02`. The sample contains Ga(As,Sb) nanowires measured by Dr. Mikel Gomez from PDI together with Dr. Gunnar Gusch at University of Cambridge and follows an analysis performed in Mikel's doctoral thesis)*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ca8d565-84be-4b8c-b7f5-dfdf008b0cf8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "streak02 = hs.load(\"../demo-files/CLEBIC2025/GaAsSb_streak_image.hspy\")\n",
+    "streak02.axes_manager"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39f5d52a-e6fb-4b80-9b59-7c7a60466c17",
+   "metadata": {},
+   "source": [
+    "Convert spectral axis from wavelength to energy *(works also for streak images since LumiSpy 0.3)*:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "91124e9a-c00d-4d05-9211-bd8f833c325f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "streak02.to_eV(inplace=True)\n",
+    "streak02.axes_manager"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b082736b-f3c9-498b-9c7a-d79ea6be2d5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "streak02.plot(cmap = 'turbo')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2aca5b52-3e7c-4eef-8270-31cd3a7aac8f",
+   "metadata": {},
+   "source": [
+    "Use the `sum()` wrapper function, we can add up the map in the time axis and get a summed spectrum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3522dac7-1014-4b3b-b128-938e08e49aff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "streak02.sum(axis='Time').plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9a07361-deb9-4444-a34a-ebf29be903cc",
+   "metadata": {},
+   "source": [
+    "We identify two convolved peaks, therefore if we want to separate the decays of these two peaks, we cannot simply take vertical slices. \n",
+    "\n",
+    "Instead, we fit two Gaussians for every pixel along the time axis of the streak image to deconvolute the two contributions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ce5985b-d991-4afb-8171-d31ea332d112",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# We convert our time axis to be a navigational one, \n",
+    "# therefore our signal becomes a list of spectra \n",
+    "# signals instead of on a image\n",
```

Comment:
```suggestion
    "# We convert our time axis to a navigation axis, \n",
    "# thus our signal becomes a series of spectra \n",
    "# (1D signal) instead of an image (2D Signa):)\n",
```